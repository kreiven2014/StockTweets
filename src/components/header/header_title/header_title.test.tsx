/* REACT */
import React from "react";

/* MODULES */
import renderer from "react-test-renderer";

/* CUSTOM MODULES */
import HeaderTitle from "./header_title";

describe("HeaderTitle tests", () => {
  test("renders correctly", (): void => {
    const tree = renderer
      .create(<HeaderTitle textKey="forgot_password:title" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
