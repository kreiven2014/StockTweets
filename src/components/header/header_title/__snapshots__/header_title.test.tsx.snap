// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderTitle tests renders correctly 1`] = `
<Text
  allowFontScaling={false}
  ellipsizeMode="tail"
  numberOfLines={1}
  style={
    Array [
      Object {
        "color": "#000000",
        "fontSize": 24,
        "letterSpacing": 0.6,
        "lineHeight": 27.5,
        "textAlign": "left",
      },
      undefined,
      Object {
        "color": "#2F323D",
        "fontSize": 41,
        "fontWeight": "bold",
        "lineHeight": 55,
      },
    ]
  }
>
  forgot_password:title
</Text>
`;
